Makefile

```diff
@@ -121,10 +134,13 @@ help:
 
 .PHONY: couch
 # target: couch - Build CouchDB core, use ERL_OPTS to provide custom compiler's options
-couch: config.erl
+couch: config.erl derived set_otp_vsn
 	@COUCHDB_VERSION=$(COUCHDB_VERSION) COUCHDB_GIT_SHA=$(COUCHDB_GIT_SHA) $(REBAR) compile $(COMPILE_OPTS)
 	@cp src/couch/priv/couchjs bin/
 
+# target: set_otp_vsn - ensure that vsn field in OTP app is set appropriately
+set_otp_vsn:
+	@sed -i$(sed v < /dev/null 2> /dev/null || echo -n " ''")  -e "s/{vsn, git}/${SUB_VSN}/" $(shell find ./src -type f -name \*.app.src)
```

Comment:
The crazy subshell here is to detect if we are using GNU sed (which does not require a file extension to be specified for `-i`, or BSD sed, which _does_ require a file extension (which we then set to null).